src/search/landmarks/exploration.cc

```diff
@@ -124,11 +124,19 @@ void Exploration::build_unary_operators(const OperatorProxy &op) {
     }
 }
 
-// heuristic computation
-void Exploration::setup_exploration_queue(const State &state,
-                                          const vector<FactPair> &excluded_props,
-                                          const unordered_set<int> &excluded_op_ids,
-                                          bool use_h_max) {
+/*
+  This function initializes the first layer of the relaxed planning task and the
+  priority queue for unrolling the relaxed task graph. Unary operators are
```

Comment:
I would change the first sentence to:

This function initializes the priority queue and the information associated with propositions and unary operators for the relaxed exploration. Unary operators that are not allowed to be applied due to exclusions are marked by setting their h_add_cost to -2 (sigh).

Reply:
Similarly, the h_add_cost of excluded unary propositions is set to -2.